presto-main/src/test/java/io/prestosql/execution/TestCallTask.java

```diff
@@ -135,4 +152,14 @@ public static void testingMethod(Long id)
     {
         invocations.add(id);
     }
+
+    private static class DenyProcedureAccessControl
```

Comment:
You can use generic DenyAll since you're going to verify the exception message anyway